common/logisticspipes/gui/GuiGuideBook.java

```diff
@@ -0,0 +1,312 @@
+package logisticspipes.gui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GLSync;
+import scala.actors.threadpool.Arrays;
+
+import logisticspipes.LPConstants;
+import logisticspipes.items.ItemGuideBook;
+import logisticspipes.utils.string.StringUtils;
+
+public class GuiGuideBook extends GuiScreen {
+
+	/**
+	 * Z Levels:
+	 * Title and Buttons: 15
+	 * Background frame : 10
+	 * Text             : 5
+	 * Background       : 0
+	 * Background tint  : -5
+	 */
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	private GuiButton nextPageBtn, prevPageBtn, slider;
+	private int itemSlot;
```

Comment:
This never gets set anywhere, does it? Add it as a constructor parameter

Reply:
I ended up not really needing it at all